pkg/command/config.go

```diff
@@ -81,9 +83,10 @@ var getConfigCmd = &cobra.Command{
 }
 
 var setConfigCmd = &cobra.Command{
-	Use:   "set <path> <value>",
-	Short: "Set config values at the given path",
-	Long:  "Set config values at the given path. path values: [unstable-versions, cli.edition, features.global.<feature>, features.<plugin>.<feature>, env.<variable>]",
+	Use:               "set <path> <value>",
+	Short:             "Set config values at the given path",
+	Long:              "Set config values at the given path. path values: [unstable-versions, cli.edition, features.global.<feature>, features.<plugin>.<feature>, env.<variable>]",
```

Comment:
(not related to this PR but) I feel we should remove these 2 additional values from the help text as those are not applicable anymore.

```suggestion
	Long:              "Set config values at the given path. path values: [features.global.<feature>, features.<plugin>.<feature>, env.<variable>]",
```

Reply:
I agree with you.  However, since we have plans to clean that documentation throughly as a separate effort, I'll leave it for another PR.